### DISCUSSION TITLE
Add prop to add className to prefix or suffix

### DISCUSSION THREAD

**Discussion by hexadeciman** [barvian/number-flow#83] [Ideas]
I would be super nice if NumberFlow element exposed 2 props:
`suffixClassName` and `prefixClassName` which would allow us to style them separately.

Can I create a PR for that ?

**Comment by barvian** [comment]
Hi, they can be styled separately using the `::part` syntax ([docs](https://number-flow.barvian.me/#styling)): https://codesandbox.io/p/sandbox/number-flow-react-forked-r4j23r?workspaceId=ws_SANn7Fxiketop95tKwKpeJ 👋
